packages/server/graphql/public/mutations/acceptTeamInvitation.ts

```diff
@@ -51,8 +51,6 @@ const acceptTeamInvitation: MutationResolvers['acceptTeamInvitation'] = async (
   const {invitation} = invitationRes
   const {meetingId, teamId, invitedBy: inviterId} = invitation
   const acceptAt = invitation.meetingId ? 'meeting' : 'team'
-  const meeting = meetingId ? await dataLoader.get('newMeetings').load(meetingId) : null
-  const activeMeetingId = meeting && !meeting.endedAt ? meetingId : null
```

Comment:
@mattkrick I'd like to make this change as part of a fix for `/invitation-required` not redirecting when `meetingId` points to a meeting that has already ended - are there any negative side effects I might not be seeing here?